heron/spi/tests/java/com/twitter/heron/spi/utils/ShellUtilsTest.java

```diff
@@ -125,7 +117,11 @@ public void testGetProcessBuilder() throws IOException {
 
     // Process running normally
     Process p = pb.start();
-    wait(10, TimeUnit.MILLISECONDS);
+    try {
+      p.waitFor(2, TimeUnit.SECONDS);
+    } catch (InterruptedException e) {
```

Comment:
It would be better to| declare this in the throws clause and not handle it, since it should trigger a test failure.